packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -363,6 +410,103 @@ contract OPContractsManager is ISemver {
         return output;
     }
 
```

Comment:
I'm wondering if there are any other input validations we should be doing on this method, or can/should we just trust that they are handled by the proofs contracts themselves?

Reply:
OK. can resolve this if/when they are listed in your opd issues.

Reply:
tracked here: https://github.com/ethereum-optimism/optimism/issues/13698